javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  
+    studentName === "" ? console.log("Please add a name!"):undefined;
+
+    if (studentName == "Queen"  || studentName === "queen"){
+        class07Students.push("Queen");
+        return;
+    } else if(class07Students.length >= 6 && studentName !== "Queen"){
```

Comment:
Great usage of JS local scope.
You have already passed the condition (if student name === queen => push to the array), so you don't need to mention it again.